js/btcmarkets.js

```diff
@@ -85,6 +86,104 @@ module.exports = class btcmarkets extends Exchange {
         });
     }
 
+    async fetchTransactions (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+            'limit': limit,
```

Comment:
The `limit` and `since` are both optional in the signature of this method (`since = undefined, limit = undefined`), so we need to send them only if the user has specified them. Otherwise we should not send them and that falls back to exchange's defaults.